index.Rmd

```diff
@@ -0,0 +1,181 @@
+                            pattern = "Less than 15",
+                            replacement = "15 and younger")) %>% 
+  mutate(Type = str_replace(string = Type,
+                            pattern = "\\(People\\)",
+                            replacement = "\\(Births)\\")) %>%
+  pivot_wider(names_from = Type, values_from = Births)
+
+
+
+births_long <- 
+  births_clean %>%
+  
+  pivot_longer(!Years, names_to = "year", values_to = "children_born") %>% 
+  rename(Type = Years, Year = year, Births = children_born) %>% 
+  select(Year, Births, Type) %>% 
+  mutate(Year = as.numeric(Year)) %>% 
+  arrange(Year) %>%
+  mutate(Type = str_remove_all(string = Type,
+                               pattern = "\\(People\\)")) %>% 
+  mutate(Type = str_replace(string = Type,
+                            pattern = "Births ",
+                            replacement = "Total births"
+                            )) %>% 
+  mutate(Type = str_replace(string = Type,
+                            pattern = "Less than 15",
+                            replacement = "15 and younger"))
+```
+
+As we take a look at the following graphs the aim is to discover a trend in birth rates based on the average life expectancy and ethnicity of mothers. Are there clear trends? Do the trends have anything to with each other? Or, are there other elements that could be affecting the trends that we see?
+
+
+# Total births per year
+
+
+First, we take a look at the birth rates for all women in the US. This shows a peak that occurs in 2007 and then a sharp decline in births afterwards. 
+
+```{r}
+birth_graph(total)
+```
+
+
+# Change in birth rates based on the Mother's age
+
+Here we can see that the 35-39 year old age group has a clear increase in birth rates. It crosses paths with the 15-19 year old age group during the year 2000 and continues to increase as the 15-19 year old age group declines. The 20-24 year old age group also mimics the same steady decline that the 15-19 year old age group follows. There is also a uniform peak at all age levels during the year 2007.
+
+```{r}
+birth_graph(age)
+```
+
+
+# Life Expectancty and Ethnicity
+
+From this graph it is clear that life expectancy has risen over the years. This was expected as modern medicine has developed and become more wide spread (another study for another time). There is a significant difference between the life expectancies of White, Black and Hispanic women, with Hispanic women having the highest life expectancy and Black women having the lowest. 
+
+```{r}
+types_of_women <- c("All Women (Age)", "White Women (Age)", "Black Women (Age)", "Hispanic Women (Age)", "White non Hispanic Women (Age)", "Black non Hispanic Women (Age)")
+
+life_double <-
+  life_expect %>% 
+  select(-c('1980', '1981', '1982', '1983', '1984', '1985', '1986', '1987', '1988', '2020')) %>% 
+  slice(-c(26:33)) %>% 
+  filter(!Years %in% c('Men (Years)', 'By gender', 'By race and ethnicity')) %>% 
+  slice(-c(1, 3, 5, 7, 9, 11)) %>% 
```

Comment:
I definitely recommend an alternate way of doing this that is more programmatic / more interpretable.